src/Digdir.Domain.Dialogporten.WebApi/appsettings.Development.json

```diff
@@ -76,6 +76,8 @@
     "UseLocalDevelopmentCompactJwsGenerator": true,
     "DisableCache": true,
     "DisableAuth": true,
-    "UseInMemoryServiceBusTransport": true
+    "UseInMemoryServiceBusTransport": true,
+    "DisableSubjectResourceSyncOnStartup": false,
```

Comment:
La til `DisableSubjectResourceSyncOnStartup` og satte denne til `false`siden den er kritisk; hvis du setter opp et nytt miljø må du ha denne enabled for at liste-auth skal virke. Denne er i motsetning til PolicyInformation ikke noe som innebærer mye nettverkstrafikk (etter første synk). 

Det er kanskje litt uheldig både med en dobbel negativ (disable + false) mtp kognitiv load, og et avvik ift alle de andre localdev-settingsene som alle er default true. Noen options:

1. La denne være false som default
2.  La den være true som default, og dokumentere at denne må enables (eller janitor kjøres) ved etabliering av nytt devmiljø
3. Endre den til `EnableSubjectResourceSynOnStartup` og la den være true som default
4. Flippe alle `Disable*` settings til `Enable*` (og tilsvarende flippe default-verdier) for å unngå double negatives. Det vil da som i 1 være ulike verdier.  

Heller nok mot 1, alternativt 4 (men den burde være en egen PR)